docs/control-structures.rst

```diff
@@ -2,23 +2,22 @@
 Expressions and Control Structures
 ##################################
 
-.. index:: ! parameter, parameter;input, parameter;output, parameter;multiple
+.. index:: ! parameter, parameter;input, parameter;output, function parameter, parameter;function, return variable, variable;return, return
 
-Input Parameters and Output Parameters
-======================================
+Function Parameters and Return Variables
+========================================
 
-As in Javascript, functions may take parameters as input;
-unlike in Javascript and C, they may also return arbitrary number of
-parameters as output.
+As in JavaScript, functions may take parameters as input. Unlike in JavaScript
+and C, functions may also return an arbitrary number of values as output.
```

Comment:
possibly stick to "variables" (instead of "values", as in rest of document)

Reply:
I would argue that the stuff you return are values, but you do so with the help of return variables.

Reply:
I don't know how you return stuff. The issue here is "Consistency of Documentation". You introduce "Return Variables", and use the term "Variables" in the rest of the document, so "Values"  feels a bit inconsistent at this point.

Reply:
Just went through the whole document. It seems the easiest would be to to merge this as-is, and then

- Clarify difference of "Return Value" and "Return Variable" (within the document)
- use only one(!) term throughout the document/documentation (personally I would prefer "Return Value", though me not knowing if its technically the correct term).

Reply:
To me, "return value" and "return variable" are actually two different things.

Reply:
Of course its different, we agree here. I just mentioned the need to clarify this.

follow-up: #5425